lib/ansible/modules/source_control/gitlab_service.py

```diff
@@ -0,0 +1,377 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018-2019, Raphaël Droz <raphael@droz.eu>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: gitlab_service
+short_description: Creates, updates or deletes GitLab services
+version_added: "2.8"
+description:
+  - Setup or delete GitLab integration services.
+author: "Raphaël Droz (@drzraf)"
+requirements:
+  - python >= 2.7
+  - python-gitlab
+extends_documentation_fragment:
+  - auth_basic
+options:
+  api_token:
+    description:
+      - GitLab personal token for logging in (preferred method)
+    default: null
+  project:
+    description:
+      - The ID or path of the project (urlencoded or not)
+    required: true
+    default: null
+  active:
+    description:
+      - Not yet supported (U(https://gitlab.com/gitlab-org/gitlab-ce/issues/41113))
+    type: bool
+    default: 'yes'
```

Comment:
Quote are not required here

```suggestion
    default: yes
```